File: CS/13.1/WinSolution.Module.Win/Updater.cs
using System;
using DevExpress.ExpressApp.Updating;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace WinSolution.Module.Win {
    public class Updater : ModuleUpdater {
        public Updater(DevExpress.ExpressApp.IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            Note note = ObjectSpace.CreateObject<Note>();
            note.Text = "test";
            ObjectSpace.CommitChanges();
        }
    }
}

File: VB/WinSolution.Module.Win/Updater.vb
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

Namespace WinSolution.Module.Win
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As DevExpress.ExpressApp.IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim note As Note = ObjectSpace.CreateObject(Of Note)()
			note.Text = "test"
			ObjectSpace.CommitChanges()
		End Sub
	End Class
End Namespace
